flutter/test/navigation/sentry_display_widget_test.dart

```diff
@@ -0,0 +1,115 @@
+// ignore_for_file: invalid_use_of_internal_member
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+import 'package:sentry/src/sentry_tracer.dart';
+import 'package:sentry_flutter/src/integrations/integrations.dart';
+import 'package:sentry_flutter/src/navigation/sentry_display_widget.dart';
+
+import '../fake_frame_callback_handler.dart';
+import '../mocks.dart';
+
+void main() {
+  PageRoute<dynamic> route(RouteSettings? settings) => PageRouteBuilder<void>(
+        pageBuilder: (_, __, ___) => Container(),
+        settings: settings,
+      );
+
+  late Fixture fixture;
+
+  setUp(() {
+    fixture = Fixture();
+  });
+
+  testWidgets('SentryDisplayWidget reports manual ttid span after didPush',
+      (WidgetTester tester) async {
+    final currentRoute = route(RouteSettings(name: 'Current Route'));
+
+    await tester.runAsync(() async {
+      fixture.navigatorObserver.didPush(currentRoute, null);
+      await tester.pumpWidget(fixture.getSut());
+      await fixture.navigatorObserver.completedDisplayTracking?.future;
+    });
+
+    final tracer = fixture.hub.getSpan() as SentryTracer;
+    final spans = tracer.children.where((element) =>
+        element.context.operation ==
+        SentrySpanOperations.uiTimeToInitialDisplay);
+
+    expect(spans, hasLength(1));
+    final ttidSpan = spans.first;
+    expect(ttidSpan.context.operation,
+        SentrySpanOperations.uiTimeToInitialDisplay);
+    expect(ttidSpan.finished, isTrue);
+    expect(ttidSpan.context.description, 'Current Route initial display');
+    expect(ttidSpan.origin, SentryTraceOrigins.manualUiTimeToDisplay);
```

Comment:
`m`: I think adding asserts for the duration of the ttidSpan would improve this test.

Reply:
Same comment as above